web/sites/default/config/webform.webform.contact_elected.yml

```diff
@@ -486,7 +566,7 @@ settings:
   draft_pending_multiple_message: ''
   confirmation_type: page
   confirmation_title: 'Thank you for your submission'
-  confirmation_message: "We appreciate your feedback.<br />\r\n<br />\r\nIf you requested a meeting with an elected official, and the request is approved, someone will reach out to you to schedule the meeting."
+  confirmation_message: "We appreciate your feedback.<br />\r\n<br />\r\nIf you requested a meeting with an elected official, and the request is approved, someone will reach out to you to schedule the meeting.<br />\r\n<br />\r\n&nbsp;"
```

Comment:
Is there a way we could dynamically set this message based on what they select? I would imagine a minority of submissions request a meeting, so it seems odd to call it out this way unless they actually made that request. 🤔